backend/static/js/mapview.js

```diff
@@ -67,13 +68,18 @@ mapViewPage = {
         }
         let tileLayerURL = this.options.tileURL
         let tileLayerOptions = {
-            attribution:  this.options.mapAttribution + '<a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a> | Icons by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">flaticon.com</a>',
+            attribution:  this.options.mapAttribution,
             maxZoom: 18,
             id: 'mapbox/streets-v11',
             tileSize: 512,
             zoomOffset: -1,
             preferCanvas: true,
-          }
+        }
+        if (this.options.tileServer == 'mapbox') {
+          tileLayerOptions.attribution += '| <a href="https://www.mapbox.com/map-feedback/" target="_blank">Improve this map</a> ';
+        }
+        tileLayerOptions.attribution += '| Icons by <a href="https://www.flaticon.com/authors/freepik" title="Freepik">Freepik</a> from <a href="https://www.flaticon.com/" title="Flaticon">flaticon.com</a>';
```

Comment:
These icons are not used anymore, they were added back in efd1d6a and looked like this:
![Screenshot from 2020-07-08 23-27-35](https://user-images.githubusercontent.com/7229585/86972612-82517e00-c173-11ea-9c5f-01374307eef0.png)
but are not used anymore, so this can be removed.